SOC/common/simulation.py

```diff
@@ -216,6 +216,26 @@ def animate(i):
             display(HTML(anim.to_html5_video()))
         else:
             return anim
+
+    def save(self, file_name = 'sim'):
+        """ serialization of object and saving it to file"""
+
+        root = zarr.open_group('state/' + file_name + '.zarr', mode = 'w')
+        values = root.create_dataset('values', shape = (self.L_with_boundary, self.L_with_boundary), chunks = (10, 10), dtype = 'i4')
+        values = zarr.array(self.values)
+        #data_acquisition = root.create_dataset('data_acquisition', shape = (len(self.data_acquisition)), chunks = (1000), dtype = 'i4')
```

Comment:
To jest lista, fakt, i to się sypnie. Ale `data_acquisition` jest listą słowników rzeczy czysto pythonowych (stringi, inty, floaty). Możemy więc wpisać je do `attr`: `zarr` powinien wewnętrznie ją zserializować używając jsona :)